### DISCUSSION TITLE
How to set round corners ?

### DISCUSSION THREAD

**Discussion by SCDefence** [ImaginativeShohag/Why-Not-Image-Carousel#47] [Q&A] [answered]
Is there a function to set round corners or i should have to use a mask?

**Comment by ImaginativeShohag** [comment, marked as answer]
No, there is no function to create round corners.

You can make round corners in many ways. See the samples from [here](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/tree/master/sample). You can wrap your view using `MaterialCardView` and change the `app:cardCornerRadius` value to round the corners.

**Comment by ricardo-mrgd** [comment]
Hi, you can post solution in JAVA?

**Reply by ImaginativeShohag** [reply]
Answered in your separate discussion. :)
